### PR TITLE
Describe Observe vs ObserveAt

### PR DESCRIPTION
**Pull request by markus-da**
Fixes https://github.com/digital-asset/daml-finance/issues/797

### CODE REVIEW

**Review by brian-weir — APPROVED** [review]

File: docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst [hunk 1]
```diff
@@ -203,7 +203,19 @@ The Observation Parameter
 =========================
 
 ``o`` is the type used to represent ``Observation``\ s, which are time-dependent quantities that can
-be observed at any given time (such as the “EURUSD” exchange rate in the example above).
+be observed at any given time (such as the “EURUSD” exchange rate in the example above). There are
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst:206]: This section is meant to describe the type parameters involved in a claim, so the description of the specific observation combinators should probably live in a separate place.

There are also other combinators in addition to `Observe` and `ObserveAt`, such as `Const` and the algebraic combinators `+` `*`

File: docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst [hunk 2]
```diff
@@ -203,7 +203,19 @@ The Observation Parameter
...
+two ways of creating an observation in the library:
+
+- :ref:`Observe <constr-contingentclaims-core-observation-observe-30391>` is the original Claims
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst:209]: The paper doesn't really tell you anything about how to model an observation, so I wouldn't say it is the original operator :)

File: docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst [hunk 3]
```diff
@@ -203,7 +203,19 @@ The Observation Parameter
...
+  node described in the paper. It does not take a time parameter. Instead, you need to put a
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst:210]: You don't really need a `When` node in front of it. 

The observable is simply evaluated at the contract's acquisition time. 

The `When` is a way to fix the acquisition time, but in general an `Anytime` node also fixes an acquisition time

File: docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst [hunk 4]
```diff
@@ -203,7 +203,19 @@ The Observation Parameter
...
+  ``when`` node in front of it. For simple claims this is quite straightforward, but for more
+  complex payoffs it can be difficult to correctly construct the claim.
+- :ref:`ObserveAt <constr-contingentclaims-core-observation-observeat-8418>` is similar, but it
+  takes a time parameter in addition. This makes it significantly easier to use, especially for
+  complex expressions combining several features (e.g. multiple fixing dates, FX adjusted notional,
+  amortizing notional, etc). It is also safer from a lifecycling point of view, since the time
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst:216]: Luciano wouldn't agree with this statement, arguably `ObserveAt` is less safe :)

**Review by matteolimberto-da — CHANGES_REQUESTED** [review]
I think the explanation is in the wrong place and not very clear from the perspective of a new user. 

We shouldn't say "use this, not that" but provide examples to let the user understand the value of each combinator. 

Happy to discuss how we can make this more user-friendly

File: docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst (hunk 1 above)
Comment (markus-da) [reply docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst:206]: > so the description of the specific observation combinators should probably live in a separate place.

Where do think this belongs? In one of the existing places or in a new one?

**Comment by matteolimberto-da** [comment]
Supplanted by https://github.com/digital-asset/docs.daml.com/pull/397

**Pull request closed without merging**